File: python/example4.py
import matplotlib.pyplot as plt
import numpy as np
from pydraw import *
from scipy import signal

# Example of different waves

drawPaper(x_lim_min=-1,x_lim_max=10,y_lim_max=2.5,y_lim_min=-2.5,
          x_label='t / ms',y_label='u / u\u209C\u1D63',
          x_tick_spacing=2,
          y_tick_multiplicator=0.5)

# create a sine wave
x = np.arange(0,9.5,0.01)
y = 2*np.sin(x)
plt.plot(x,y)

# create a sawtooth wave 
y2 = 2*signal.sawtooth(x)
plt.plot(x,y2)

# create a square wave
y3 = 2*signal.square(x)
plt.plot(x,y3)

plt.savefig("example4_2.png",bbox_inches="tight")
plt.show() 

File: matlab/example4.m
% Example to draw a fully customized paper with matlab tag/value system

paper = drawPaper('x_scale',1,'y_scale',0.5,'x_cm',12,'y_cm',7,'x_cm_max',11,'y_cm_max',6,'x_cm_min',1,'y_cm_min',1,'x_cm_orig',1,'y_cm_orig',4,'x_label','t/ms','y_label','u / utr');

% create a sine wave
x = linspace(0,9.5);
y = sin(x);
drawFunction(paper,x,y);
% create a sawtooth wave
y2 = sawtooth(x);
drawFunction(paper,x,y2,'color','green');
% create a squre wave
y3 = square(x);
drawFunction(paper,x,y3,'color','red');
